evm-bridging/src/BridgedTopShotMoments.sol

```diff
@@ -104,10 +120,90 @@ contract BridgedTopShotMoments is
         override(ERC721Upgradeable, ERC721EnumerableUpgradeable)
         returns (address)
     {
+        // Add the beforeTokenTransfer hook
+        _beforeTokenTransfer(_ownerOf(tokenId), to, tokenId);
```

Comment:
`_beforeTokenTransfer` hook was removed from the v5 of OpenZeppelin libraries so we need to add it here in the `_update` method, which is called when transferring, minting, and burning tokens